README.md

```diff
@@ -76,7 +76,7 @@ On Ubuntu Linux 22.04 (Jammy):
 
 ```shell
 git submodule update --init --recursive
-sudo apt-get install build-essential m4 openjdk-11-jdk libfmt-dev libgmp-dev libmpfr-dev pkg-config flex bison z3 libz3-dev maven python3 python3-dev cmake gcc g++ clang-14 lld-14 llvm-14-tools zlib1g-dev libboost-test-dev libyaml-dev libjemalloc-dev
+sudo apt-get install build-essential m4 openjdk-17-jdk libfmt-dev libgmp-dev libmpfr-dev pkg-config flex bison z3 libz3-dev maven python3 python3-dev cmake gcc g++ clang-14 lld-14 llvm-14-tools zlib1g-dev libboost-test-dev libyaml-dev libjemalloc-dev
```

Comment:
it's a little weird telling people to install java 17 when we in fact support Java 11 or 17. I feel like this needs to be made a little more explicit in the places in the Readme we're modifying here.

Reply:
We actually already have language in the README indicating that we support "11 or greater", in the section where we elaborate on the Java install instructions.

The stanza here is the "short version" instruction which is intended for users to copy-and-paste to install K's dependencies. The change here doesn't break the copy-paste run, but we can revert to 11 if you'd prefer.  In the section where we elaborate on how to install Java, I reverted the example instruction to version 11 to match the text.